cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -272,6 +273,13 @@ func gitIndex(c gitIndexConfig, o *indexArgs) error {
 
 	debug.Printf("successfully fetched git data for %q (%d commit(s)) in %s", o.Name, successfullyFetchedCommitsCount, fetchDuration)
 
+	logger.Info("successfully fetched git data",
+		zap.String("name", o.Name),
+		zap.Uint32("id", o.RepoID),
+		zap.Int("commits_count", successfullyFetchedCommitsCount),
+		zap.Int64("duration_ms", fetchDuration.Milliseconds()),
+		zap.String("duration_str", fetchDuration.String()),
```

Comment:
FYI there is a `zap.Duration`. Also why log both, maybe its good enough to just do zap.Duration which will then marshal the duration for machines or humans depending on the output mode.

```suggestion
		zap.Duration("duration", fetchDuration),
```

Reply:
Updated, I'll add a comment to the updated file showing the formatted log output